test/client/fixtures/utils/dom-test.js

```diff
@@ -192,6 +192,29 @@ if (window.PerformanceNavigationTiming) {
     });
 }
 
+test('isArrayBuffer', function () {
+    ok(domUtils.isArrayBuffer(new ArrayBuffer(0)), 'ArrayBuffer');
+    ok(!domUtils.isArrayBuffer(void 0), 'undefined');
+});
+
+test('isArrayBufferView', function () {
+    ok(domUtils.isArrayBufferView(new Int8Array(0)), 'Int8Array');
+    ok(domUtils.isArrayBufferView(new Uint8Array(0)), 'Uint8Array');
```

Comment:
Why does we test all Array like data types? We need test only for two cases: true and false